message-service/src/main/java/sjchat/messages/MessageService.java

```diff
@@ -140,6 +115,11 @@ public void getChat(GetChatRequest req, StreamObserver<GetChatResponse> response
 
   @Override
   public void createChat(CreateChatRequest req, StreamObserver<CreateChatResponse> responseObserver) {
+    if (req.getTitle() == null || req.getTitle().length() == 0) {
+      responseObserver.onError(new Exception("Title is empty or missing"));
+      responseObserver.onCompleted();
+    }
```

Comment:
You can refactor this validation code into a method like `validateNonEmptyTitle(req, responseObserver);` (DRY principle).

I'm not 100 % sure that is a good idea though, it might not be obvious that `validateNonEmptyTitle()` will return on failure. The validator should also maybe be located in a higher "layer", does it exist any standard place to put validation for gRPC arguments? Some kind of validation middleware perhaps?